src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/__artifactId__-servlet.xml

```diff
@@ -25,6 +25,9 @@ ${shogun2-parent-package}.*.web" )
 			serialization -->
 		<mvc:message-converters>
 			<beans:bean
+				class="org.springframework.http.converter.ByteArrayHttpMessageConverter">
+			</beans:bean>
```

Comment:
What exactly is this doing? What would be the default behaviour if we would not use it?